src/main/kotlin/it/pagopa/ecommerce/payment/requests/repositories/PaymentRequestInfo.kt

```diff
@@ -18,4 +18,5 @@ constructor(
   val dueDate: String?,
   val paymentToken: String?,
   val idempotencyKey: IdempotencyKey?,
-)
+  val transferList: List<PaymentTransferInfo>?,
```

Comment:
This is expected to be `null` when a notice code is verified but not activated right?